recipe/meta.yaml

```
@@ -26,30 +25,34 @@ requirements:
     - setuptools
   run:
     - python
+    - dnspython >=1.16.0,<3.0.0
```

```suggestion
    - dnspython >=1.16.0,<3.0.0
  run_constrained:
    - pymongo-auth-aws <2.0.0
    - pymongocrypt >=1.6.0,<2.0.0
    - winkerberos >=0.5.0 # [win]
    - pyopenssl >=17.2.0
    - requests <3.0.0
    - cryptography >=2.5
    - service_identity >=18.1.0
```
https://github.com/mongodb/mongo-python-driver/blob/4.6.3/pyproject.toml#L46